pihole

```diff
@@ -572,8 +572,8 @@ fi
 
 # Check if the current user is not root and if the command
 # requires root. If so, exit with an error message.
-if [[ $EUID -ne 0 && need_root -eq 1 ]];then
-  echo -e "  ${CROSS} The Pi-hole command requires root privileges, try:"
+if [[ $EUID -ne 0 && ${need_root} ]];then
```

Comment:
This does work but I think we should be explicit with what we are `test`ing for, so `test` for `need_root` to be set/non-empty
```suggestion
if [[ $EUID -ne 0 && -n "${need_root}" ]];then
```